### ISSUE TITLE
Move all notification templates from project resources into embedded resources as solution files

### ISSUE THREAD

**Issue by tatarincev** [VirtoCommerce/vc-module-order#74]
**The problem:**
Notification templates included as project resources are difficult to manage and see changes (version control) and it gives a very bad example for custom projects.

**Solution:**
Move all templates from project resource
![image](https://user-images.githubusercontent.com/7566324/50819679-72b16900-1333-11e9-87c8-b49abcb00343.png)
to special folder in project `VirtoCommerce.OrderModule.Data/Notifications/Templates`

Set for a file `Build action` as `Embedded resource` 
![image](https://user-images.githubusercontent.com/7566324/50819926-11d66080-1334-11e9-8401-0ab94cc6e0ef.png)

and use by this code
```C#
 var assembly = GetType().Assembly;
  notificationManager.RegisterNotificationType(() => new OrderEmailNotification(_container.Resolve<IEmailNotificationSendingGateway>())
            {
...
                NotificationTemplate = new NotificationTemplate
                {
                    Body = assembly.GetManifestResourceStream("VirtoCommerce.OrderModule.Data.Notifications.Templates.NotifocationTemplateBody.html").ReadToString(),
                    Subject =assembly.GetManifestResourceStream("VirtoCommerce.OrderModule.Data.Notifications.Templates.NotifocationTemplateSubject.html").ReadToString(),
...
                }
            });
```

**Comment by yecli** [comment]
Found that html subject templates contain \r\n because of autoformat on save in VS. This leads to errors when sending all these emails with real mail provider - \r\n cannot be the part of Subject.
Need to remove trailing line feeds and to save all templates again outside of VS.

**Comment by yecli** [comment]
Released in [v2.17.22](https://github.com/VirtoCommerce/vc-module-order/releases/tag/v2.17.22).
